medusa/backup_cluster.py

```diff
@@ -68,9 +68,9 @@ def orchestrate(config, backup_name_arg, seed_target, stagger, enable_md5_checks
 
         logging.debug('Emitting metrics')
 
-        logging.info('Backup duration: {}'.format(backup_duration.seconds))
+        logging.info('Backup duration: {}'.format(backup_duration.total_seconds))
```

Comment:
**Question:** shouldn't this be `.total_seconds()` as well?

Reply:
It should, yes, my bad…